### PR TITLE
_pkginfo: Improve error handling

### PR DESCRIPTION
**Pull request by jayvdb**
Try to extract error message from pkginfo.
Also handle no response from pkginfo, and handle
JSON decode errors.

### CODE REVIEW

File: dephell_setuptools/_pkginfo.py [hunk 1]
```diff
@@ -17,6 +17,15 @@ def content(self) -> Dict[str, Any]:
...
+            msg_lines = result.stderr.decode().rstrip().split('\n')
+            raise RuntimeError(msg_lines[-1] if msg_lines else 'Unknown error')
+        stdout = result.stdout.decode()
+        if not stdout:
+            return {}
+
+        try:
+            content = json.loads(stdout)
+        except json.decoder.JSONDecodeError:
+            return {}
```
Comment (orsinium) [inline dephell_setuptools/_pkginfo.py]: Why this should be silenced? All exceptions from readers go into `read_setup` where they are handled by callback function.

Comment (jayvdb) [reply dephell_setuptools/_pkginfo.py]: This one was causing exceptions reading invalid json from pkginfo to be shown in the console, usually when pkginfo was emitting nothing, which isnt valid json.

Comment (orsinium) [reply dephell_setuptools/_pkginfo.py]: Good point :)

**Comment by orsinium** [comment]
Thank you. As always, great work :)

**Pull request merged**